Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -85,9 +85,9 @@ public TestConductRepository(TrappistDbContext dbContext, ITestsRepository testR
             return null;
         }
 
-        public async Task<bool> IsTestLinkExistAsync(string magicString)
+        public async Task<bool> IsTestLinkExistForTestConductionAsync(string magicString)
         {
-            var isTestLinkExist = await (_dbContext.Test.AnyAsync(x => (x.Link == magicString)));
+            var isTestLinkExist = await (_dbContext.Test.AnyAsync(x => x.Link == magicString && x.IsPaused==false ));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the unnecessary Boolean literal(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1125)